### DISCUSSION TITLE
New Ikea Tradfri bulb unsupported

### DISCUSSION THREAD

**Discussion by CzechSebastian** [Koenkk/zigbee2mqtt#7215] [Devices]
Hi,
I just bought an Ikea bulb ( LED1835C6 E12 450lm )
When I want to pair with Z2M , I have this :
![image](https://user-images.githubusercontent.com/46093860/116113338-67121980-a686-11eb-8f2f-95d8be764d88.png)
I already reset it with 6 turn on/off method but I have the same issue.
How can I resolve this ?

**Comment by Koenkk** [comment]
Should already be supported in the latest dev branch (https://www.zigbee2mqtt.io/how_tos/how-to-switch-to-dev-branch.html) or wait for the next release on 1 May

**Reply by CzechSebastian** [reply]
Incredible ! Thanks for the info !
I'll be wait for may 1st !

**Reply by CzechSebastian** [reply]
> Should already be supported in the latest dev branch (https://www.zigbee2mqtt.io/how_tos/how-to-switch-to-dev-branch.html) or wait for the next release on 1 May

Hi,
Does the new version is available?
How can I update it in home assistant?
Thanks

**Reply by Koenkk** [reply]
I've just released it, HA addon is building now, should be available in 1 hour from now.

**Reply by CzechSebastian** [reply]
Because it's weird I'm on 1.18.1 and the 1.18.2 is available since one month and no update.
Can you tell me when the new version will be ready, I'll check it and tells you.
Thanks again 🎉

**Reply by Koenkk** [reply]
I think you need to change the repo url from https://github.com/danielwelch/hassio-zigbee2mqtt to https://github.com/zigbee2mqtt/hassio-zigbee2mqtt

**Reply by CzechSebastian** [reply]
Have you a screenshot of where I do that?

**Reply by CzechSebastian** [reply]
Can I save my devices and restored to the other version?

**Reply by Koenkk** [reply]
https://github.com/zigbee2mqtt/hassio-zigbee2mqtt#installation

yes, everything is restored automatically

**Reply by Koenkk** [reply]
@ciotlosm it seems some users don't see the updates from the new repo, do you know what is going on?

**Reply by CzechSebastian** [reply]
Can juste change the url of the repo or I need to re-install a version from New repo and delete the older one?

**Reply by CzechSebastian** [reply]
That's OK I add the new repo and delete the other.
It's all good I have all my devices.
Now I just need to test my new ikea bulb

**Reply by CzechSebastian** [reply]
I already I the same issue 
<img width="942" alt="Capture d’écran, le 2021-05-01 à 10 26 34" src="https://user-images.githubusercontent.com/46093860/116785488-ce094700-aa67-11eb-9e28-29155f526163.png">
The device is unsupported

**Reply by Koenkk** [reply]
It's indeed not supported yet, `LED1835C6` is supported but I think your model is different. `LED1835C6` is a 470 lumen bulb. Is there any other model written on the box or bulb itself?

**Comment by CzechSebastian** [comment]
That's all that I have On box and bulb, can you do something ?
![IMG_7499](https://user-images.githubusercontent.com/46093860/116788734-50e6cd80-aa79-11eb-9856-4c642781d93b.JPG)
![IMG_7498](https://user-images.githubusercontent.com/46093860/116788735-53e1be00-aa79-11eb-8756-e64fc6a1f0f1.JPG)
![IMG_7497](https://user-images.githubusercontent.com/46093860/116788736-56441800-aa79-11eb-8b56-11ddd9ad263e.JPG)
![IMG_7496](https://user-images.githubusercontent.com/46093860/116788739-58a67200-aa79-11eb-8e0c-c02e178a0b2c.JPG)

**Reply by Koenkk** [reply]
Thanks, seems IKEA uses the same model for slightly different devices.

Added it now.

Changes will be available in the dev branch in a few hours from now. (https://www.zigbee2mqtt.io/how_tos/how-to-switch-to-dev-branch.html)

**Reply by CzechSebastian** [reply]
And how do you release them?
Because I don't know how use the Dev branch with home assistant addon

**Reply by CzechSebastian** [reply]
> Thanks, seems IKEA uses the same model for slightly different devices.
> 
> Added it now.
> 
> Changes will be available in the dev branch in a few hours from now. (https://www.zigbee2mqtt.io/how_tos/how-to-switch-to-dev-branch.html)

I just see that use the Dev branch with home assistant, I need to use the edge one.
Does this edge one will be updated too?

**Reply by CzechSebastian** [reply]
That okay it's work with the edge version.
Thanks a lot for you help, I imagine the release will be on June 1st?

**Reply by Koenkk** [reply]
Yes, next release will by on June 1st

**Comment by hyperion007** [comment]
Any idea when support for IKEA 'TRADFRI bulb E14 CWS 470lm' will be added?

**Comment by Dutch-Dev** [comment]
I also get "Unsupported" on this one: **TRADFRIbulbE27WSglobeopal1055lm**. Will it also be included in the June update?

**Reply by Koenkk** [reply]
`TRADFRIbulbE27WSglobeopal1055lm` will be supported in the next release

**Reply by Dutch-Dev** [reply]
Great, thank you very much!

**Comment by LaurentBl** [comment]
Hi Team,

I have unsupported with this model : TRADFRI bulb E27 CWS 806lm. I tried everything and nothing did. Any update coming up?

**Reply by Koenkk** [reply]
`TRADFRI bulb E27 CWS 806lm` is supported in z2m 1.28.0

**Reply by LaurentBl** [reply]
My bad. The update had not been done properly. It's fixed now. Thanks for the reply and apologies.

**Comment by Londonlad95** [comment]
I have some unsupported bulbs, "TRADFRI bulb E14 CWS globe 806lm" any chance of them being added?

**Reply by Koenkk** [reply]
It's supported in z2m 1.39.1

**Comment by johnsturgeon** [comment]
@Koenkk What would be the best way to get support for the `TRADFRI bulb E12 CWS globe 800lm`

https://www.ikea.com/us/en/p/tradfri-led-bulb-e12-800-lm-wireless-dimmable-color-and-white-spectrum-globe-opal-80547510/

**Reply by Koenkk** [reply]
What's the model number of this device? I expect something like `LED1XXXXXXXX`

**Reply by johnsturgeon** [reply]
LED2111G6 (PDF of box attached)

lmk if there is any more info you need. 

<img width="508" alt="image" src="https://github.com/user-attachments/assets/862b2ac1-2fdc-4f1f-96e5-2febb2d5a396">

[TRADFRI.pdf](https://github.com/user-attachments/files/17579266/TRADFRI.pdf)

**Reply by Koenkk** [reply]
Added, will be supported in tomorrows release.

**Reply by johnsturgeon** [reply]
Thank you!
